docs/README.md

```diff
@@ -16,7 +16,250 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
+    apiVersion: operator.istio.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
+    spec:
+      version: v1.22.0
+      namespace: istio-system
+      values:
+        pilot:
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1024Mi
+        global:
+          meshID: mesh1
+          multiCluster:
+            clusterName: west
+          network: network2
+    EOF
+    kubectl wait --context "${CTX_CLUSTER2}" --for=jsonpath='{.status.revisions.ready}'=1 istios/default --timeout=3m
+    ```
+
+8. Create east-west gateway on west
+
+    ```sh
+    "${ISTIOCTL_DIR}"/samples/multicluster/gen-eastwest-gateway.sh \
+        --mesh mesh1 --cluster west --network network2 | \
+        "${ISTIOCTL_DIR}"/bin/istioctl --context "${CTX_CLUSTER2}" install -y -f -
+    ```
+
+9. Expose services on west
+
+    ```sh
+    kubectl --context "${CTX_CLUSTER2}" apply -n istio-system -f \
+        "${ISTIOCTL_DIR}"/samples/multicluster/expose-services.yaml
+    ```
+
+10. Install a remote secret in west that provides access to east’s API server.
+
+    If using kind, first get the east controlplane ip and pass the `--server` option to `istioctl create-remote-secret`
+
+    ```sh
+    WEST_CONTAINER_IP=$(kubectl get nodes east-control-plane --context "${CTX_CLUSTER1}" -o jsonpath='{.status.addresses[?(@.type == "InternalIP")].address}')
+    "${ISTIOCTL_DIR}"/bin/istioctl create-remote-secret \
+      --context="${CTX_CLUSTER1}" \
+      --name=east \
+      --server="https://${WEST_CONTAINER_IP}:6443" | \
+      kubectl apply -f - --context "${CTX_CLUSTER2}"
+    ```
+
+    otherwise use
+
+    ```sh
+    "${ISTIOCTL_DIR}"/bin/istioctl create-remote-secret \
+        --context="${CTX_CLUSTER1}" \
+        --name=cluster1 | \
+        kubectl apply -f - --context="${CTX_CLUSTER2}"
+    ```
+
+11. Install a remote secret in east that provides access to west’s API server.
+
+    If using kind, first get the east controlplane ip and pass the `--server` option to `istioctl create-remote-secret`
+
+    ```sh
+    EAST_CONTAINER_IP=$(kubectl get nodes west-control-plane --context "${CTX_CLUSTER2}" -o jsonpath='{.status.addresses[?(@.type == "InternalIP")].address}')
+    "${ISTIOCTL_DIR}"/bin/istioctl create-remote-secret \
+      --context="${CTX_CLUSTER2}" \
+      --name=west \
+      --server="https://${EAST_CONTAINER_IP}:6443" | \
+      kubectl apply -f - --context "${CTX_CLUSTER1}"
+    ```
+
+    otherwise use
+
+    ```sh
+    "${ISTIOCTL_DIR}"/bin/istioctl create-remote-secret \
+        --context="${CTX_CLUSTER1}" \
+        --name=cluster1 | \
+        kubectl apply -f - --context="${CTX_CLUSTER2}"
+    ```
+
+12. Deploy sample applications and verify that you get a response from both v1 and v2 of the hellworld service.
+
+    ```sh
+    kubectl get ns sample --context "${CTX_CLUSTER1}" || kubectl create --context="${CTX_CLUSTER1}" namespace sample
```

Comment:
Can we split this step into two (i.e., east cluster and west cluster) as it would improve the readability.